modules/ocf_desktop/files/xsession/print-notify-listener

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+
+function clean_up {
+    jobs -pr | xargs pgrep -P | xargs sudo -u ocfbroker kill
```

Comment:
This seems kinda complicated, why does it need to be composed like this?

Reply:
Actually, even better: have the handler [detect when the pipe is closed](https://stackoverflow.com/questions/28315446/detect-when-reader-closes-named-pipe-fifo)

Reply:
See below in my response to your using fd 3. I came to the same conclusion. I also think a thread is a bit overkill for code as complicated as this already is.